### DISCUSSION TITLE
What is the oldest Safari version supported by Komga 1.10.3

### DISCUSSION THREAD

**Discussion by hanzerruid** [gotson/komga#1614] [Q&A]
### Steps to reproduce

I am encountering an issue with Komga 1.10.3 on my iPad running iPadOS 13.1.2 with Safari/605.1.15 AppleWebKit/605.1.15. The problem I'm facing is that the page loads blank. I can confirm that Komga works fine on both iPhone and PC platforms. According to issue #1253, I suspect this could be due to the older version of iPadOS I'm using. Could you help me clarify the oldest Safari version supported by Komga 1.10.3?

### Expected behavior

I expect Komga to load and display content as it does on other platforms (mobile and PC).

### Actual behavior

The page loads blank without displaying any content.

### Komga version

1.10.3

### Operating system

iPadOS 13.1.2

### Installation method

Docker

### Acknowledgements

- [X] I have searched the existing issues (open AND closed) and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have checked the [FAQ](https://komga.org/docs/faq).
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by DieselTech** [comment]
There are no exact version numbers when it comes to supported versions like that. It's an ever-moving target as new versions release and new features get updated.

The way it's currently defined in the project is:

```
> 0.5%
ios >= 12
last 2 versions
not dead
```

You can take that and look on [Browser List](https://browsersl.ist/#q=%3E+0.5%25%0Aios+%3E%3D+12%0Alast+2+versions%0Anot+dead) to see what it translates into. 

Even though it says it's targeting ios 12 or higher, I suspect something might of changed on vue's end recently. That or apple has stopped updating the older browsers and they are falling too far behind. What is the full user-agent of the browser? What you posted looks like it's only half the UA. [Check your UA](https://whatmyuseragent.com/). That will give us more info into the exact version of safari your on. 

Your version 13.1 works out to be 0.00% of the overall web traffic. It's highly likely that Vue (the framework where the requirements come from) decided to stop supporting something that has so little use that it registers as 0%

**Reply by hanzerruid** [reply]
Thanks for your reply. And this is the full info of my UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1.2 Mobile/15E148 Safari/604.1
It’s sad that old apple products always suffered poor service in new updates. Maybe I should update into version 15.6 in the first time.

**Reply by DieselTech** [reply]
> my UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1.2 Mobile/15E148 Safari/604.1

Your Safari version hasn't been updated in just about 2 full years. You're on 15.6.1 (August 18, 2022).

So what I think is happening here after thinking about it more:

The likely reason it's not loading even with the greater than ios 12 is because of the `not dead` statement. The number of users on 10.15.4 is so small that it doesn't even register as 0.01%. The packager isn't going to create bundles for dead OS versions. That would just bloat the app and make loading slower for users who are updated. If you have the ability to update, then you should do so. Problems loading sites will only get worse as more time goes on since you are on an OS that will never receive updates.

**Comment by miceonmars** [comment]
Workaround:
Install Tachimanga-App with Komga-Extension.
